app/Model/Product.php

```diff
@@ -34,4 +37,12 @@ public function images()
         return $this->morphMany(Image::class, 'imageable');
     }
 
+    public function tags(){
```

Comment:
hình như chưa pull master hả hậu. Nay t có sửa Product model này rồi và đã xóa cái `tags()` với cái `images`
Hiện tại là ntn cơ
public function images()
    {
        return $this->morphMany(Image::class, 'imageable');
    }

    public function taggables() {
        return $this->morphMany(Taggable::class, 'taggable');
    }